core/ledger/snapshotgrpc/snapshot_service.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package snapshotgrpc
```

Comment:
I differ with @manish-sethi suggestion here. IMO, having a bigger but smaller number of packages help in navigation, management and code readability over having a smaller but larger number of packages. On one side, we tried to reduce the number of packages and on the other side, we are increasing it.

Reply:
I tried to reduce both. Having a bigger package like `kvledger` helps in memorability and hence the productivity. I can just look at the filenames to understand what it has. There is a simple mental map of what `kvledger` has in my mind and helped many times. Having a package called `snapshotgrpc` and then file called `snapshot_service.go`, I don't know what we gain from this and personally, it annoys me while coding to move between folders, etc...

Reply:
> I don't know what we gain from this ...

I will again refer to my first [comment](https://github.com/hyperledger/fabric/pull/1683#discussion_r466001346) where I mentioned what looked odd to me when I looked at this code.